src/features/Dashboard/Dashboard.actions.ts

```diff
@@ -0,0 +1,44 @@
+import * as api from "./Dashboard.api";
+import * as constants from "./Dashboard.constants";
+import { AppThunkDispatch } from "store/types";
+import { INDEX_OF_FIRST_ELEMENT } from "constants/common";
+import { LanguageType } from "store/language/types";
+
+export const getNowPlayingMovies = (activeLanguage: LanguageType) => async (dispatch: AppThunkDispatch) => {
+  try {
+    dispatch({ type: constants.GET_NOW_PLAYING_MOVIES_START });
+
+    const nowPlayingMovies = await api.getNowPlayingMovies(activeLanguage);
+    const fiveNowPlayingMovies = nowPlayingMovies.slice(INDEX_OF_FIRST_ELEMENT, constants.NUMBER_OF_FILMS_IN_LIST);
```

Comment:
better to not use word `five` in constant name, cause when you change `NUMBER_OF_FILMS_IN_LIST ` constant this name will be wrong